gcc/rust/util/bi-map.h

```diff
@@ -48,7 +48,7 @@ template <typename K, typename V> class BiMap
     return itr->second;
   }
 
-private:
+public:
```

Comment:
typo or is that change supposed to be here?

Reply:
definitely a typo, I added an insert() method when we were trying the splitting of the BiMap, i'll fix it soon